terraform/lambda.tf

```
@@ -3,7 +3,7 @@ resource "aws_lambda_function" "lambda" {
   function_name    = "codepipeline-teams-notifications"
   role             = "${aws_iam_role.lambda.arn}"
   handler          = "index.handler"
-  source_code_hash = "${base64sha256(file("../lambda.zip"))}"
+  source_code_hash = "${base64sha256("../lambda.zip")}"
```

After testing this change, it actually needs to use the new `filebase64sha256` function for Terraform 0.12+. Using just `base64sha256("../lambda.zip")` will only hash the file path which will prevent Terraform from noticing a change.

```suggestion
  source_code_hash = "${filebase64sha256("../lambda.zip")}
```